components/alert/alert.css

```diff
@@ -0,0 +1,93 @@
+/* Only for Visualization of Example */
+body{
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.example{
+  width: 50%;
+}
+/* Only for Visualization of Example */
+
+
+/* Alert Component CSS */
+.alert{
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  font-weight: 600;
+  height: 50px;
```

Comment:
Using rem is better for responsiveness

Reply:
Thanks, have implemented the change. Good catch!